submitted lab06

nathanBonilla opened the pull request.

knizzle reviewed (commented):
@pjz987  Please review.

knizzle reviewed (changes requested):
Re-look at 'matches'; should only be a 'match' if in same index, as well.

Right now looks like:
winning ticket = [92, 26, 88, 62, 69, 38], my ticket = [94, 92, 12, 80, 38, 66]
matches = 2 --> should be 0

nathanBonilla commented:
Hi Kelsey,
From lines 46 - 49 I implemented a "Orders Strictly Matters" section where the matches are only counted when the indexes of both tickets are the same.  The lab was ambiguous on this, so I implemented multiple solutions.  I should have written it out in separate functions.  Please let me know if that function will suffice.
